modules/passkey/src/utils/webauthn.ts

```diff
@@ -0,0 +1,185 @@
+import { ethers } from 'ethers'
+import type { BigNumberish, BytesLike } from 'ethers'
+import CBOR from 'cbor'
+
+export function userVerificationFlag(userVerification: UserVerificationRequirement = 'preferred'): number {
```

Comment:
This is missing documentation comments. It used to be only used internally, but now that it is not we should document the function.

Reply:
Done 👍🏾